packages/kbn-performance-testing-dataset-extractor/src/index.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export { extractor } from './extractor';
+export * from './cli';
```

Comment:
@dmlemeshko I suggest that you use the [EditorConfig plugin for VSCode](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) so that you will get newlines when expected.

Reply:
Thank you @spalger, totally useful. I will update the PR